fab/fabfile.py

```diff
@@ -202,6 +207,7 @@ def staging():
         print ("using default branch of autostaging. you can override this "
                "with --set code_branch=<branch>")
 
+    env.force = True  # don't worry about kafka checkpoints on staging
```

Comment:
staging currently has several out of date checkpoints and figured this would be easier/less confusing than requiring people to add `force=true` every time. not sure if updating it to 30 days will fix it